scripts/exregional_save_restart.sh

```diff
@@ -110,7 +110,8 @@ n_iolayouty=$(($IO_LAYOUT_Y-1))
 list_iolayout=$(seq 0 $n_iolayouty)
 
 if [[ ${cycle_subtype} == "ensinit" ]] ; then
-  restart_prefix=${save_yyyy}${save_mm}${save_dd}.${save_hh}00${DT_ATMOS}
+  #restart_prefix=${save_yyyy}${save_mm}${save_dd}.${save_hh}00${DT_ATMOS}
```

Comment:
If line 113 is no longer needed, suggest removing it.

Reply:
save this line for future development.